src/ApplyOfflineMap.cpp

```diff
@@ -103,12 +105,23 @@ int ApplyOfflineMap(
 	std::string strVariables,
 	std::string strOutputData,
 	std::string strOutputDataList,
-	std::string strNColName, 
+	std::string strNColName,
+	std::string strInputMesh,
+	std::string strOutputMesh,
+	std::string strOverlapMesh,
+	int nPin,
+	int nPout,
 	bool fOutputDouble,
 	std::string strPreserveVariables,
 	bool fPreserveAll,
 	double dFillValueOverride,
-	std::string strLogDir
+	std::string strLogDir,
+	double lb,
```

Comment:
Why would you expect the calling function to know lower and upper bounds ? This should be deciphered from input data (`strInputData` and `strVariables`), no ? What happens when `strInputDataList` is an array ? How will this function handle it with CAAS ?

Reply:
I thought it would be known beforehand what the desired upper and lower bounds of the remapped field would be, so I included them as input parameters, but I don't think it would be too much trouble to add an option that would compute the upper and lower bounds from the input data.  And yes, I'll have to modify the fCAAS case when strInputDataList is an array.